drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -203,7 +203,15 @@
 
 .xblock--drag-and-drop--editor .items-form .item-numerical-value,
 .xblock--drag-and-drop--editor .items-form .item-numerical-margin {
-    width: 60px;
+    margin-right: 1%;
+}
+
+.xblock--drag-and-drop--editor .items-form .item-numerical-value {
+    width: 620px;
+}
+
+.xblock--drag-and-drop--editor .items-form .item-numerical-margin {
+    width: 578px;
```

Comment:
IMHO these looked better before - they seem too big now, are inconsistently sized, and they need a margin on the left - they aren't aligned with the other controls above:
![screen shot 2016-01-05 at 7 24 45 pm](https://cloud.githubusercontent.com/assets/945577/12134260/867dd6de-b3e2-11e5-8066-d0d60524dedd.png)